libfwupdplugin/fu-plugin.c

```diff
@@ -481,6 +580,12 @@ fu_plugin_device_add (FuPlugin *self, FuDevice *device)
 		return;
 	}
 
+	/* plugin is degraded, proxy to device */
+	if (fu_plugin_get_mode (self) == FU_PLUGIN_MODE_RT_DEGRADED &&
+	    fu_plugin_get_mode_text (self) != NULL) {
+		fu_device_set_update_error (device, fu_plugin_get_mode_text (self));
```

Comment:
do the devices also need to lose updatable flag?  And do all devices by the plugin get this text?

Reply:
I have a plan; more tmw.